apps/DISTAL-JIT/CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+cmake_minimum_required(VERSION 2.8.12)
```

Comment:
I'm not sure how to re-architect the build system to enable both TACO to access Legion, and the DISTAL runtime library.